specifyweb/frontend/js_src/lib/components/SkeletonLoaders/Form.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import { Skeleton } from './Skeleton';
+
+export function FormSkeleton() {
+  return (
+    <div className="w-full">
```

Comment:
consider creating one loader for Collection Object form as it's usually tall and very common, and a different, much smaller loader for all the other forms to avoid the issue Grant mention where skeleton is way larger than the form